rest-service/manager_rest/test/endpoints/test_nodes.py

```diff
@@ -385,39 +390,46 @@ def test_patch_before_put(self):
                 '1234',
                 runtime_properties={'key': 'value'}
             )
+        assert cm.exception.status_code == 404
 
-        self.assertEqual(cm.exception.status_code, 404)
-
-    def test_node_and_node_instance_properties(self):
-        _, _, _, deployment = self.put_deployment(
-            blueprint_file_name='deployment-creation-with-groups.yaml')
-        node = self.client.nodes.get(deployment.id, 'vm')
-        self.assertDictContainsSubset({
+    def test_node_actual_planned_instances(self):
+        dep2 = self._deployment('dep2', scaling_groups={
+            'group1': {
+                'members': ['vm'],
+                'properties': {'planned_instances': 2}
+            }
+        })
+        numbers = {
             'number_of_instances': 2,
             'min_number_of_instances': 2,
             'max_number_of_instances': 2,
             'planned_number_of_instances': 2,
             'deploy_number_of_instances': 2,
-        }, node)
-        instances = self.client.node_instances.list(
-            deployment_id=deployment.id
-        ).items
-        for instance in instances:
-            # test list/get/patch endpoints
-            tested_instances = [
-                instance,
-                self.client.node_instances.get(instance.id),
-                self.client.node_instances.update(instance.id)]
-            for tested_instance in tested_instances:
-                scaling_groups = tested_instance.scaling_groups
-                self.assertEqual(1, len(scaling_groups))
-                self.assertDictContainsSubset({'name': 'group1'},
```

Comment:
checking that the _instance_ has the groups, is testing instance creation in create-dep-env. I don't want to test that here.
Instead, this test is about figuring out the actual_planned_number_of_instances per node, which is something that we have specific logic for in the get nodes endpoint, so it'd be good for it to be tested